Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2
# Setting value for accesstokenAcceptedVersion, fix numbering

* Doc was missing necessary instruction to set accesstokenAcceptedVersion=2, required in app manifest when signInAudience=AzureADandPersonalMicrosoftAccount (this scenario)

* Also fixed numbering in that section, previously indent level was inconsistent

## Purpose

* Fix issue in documentation that may lead to user confusion as they go through example.
* Fix non-sequential numbering and inconsistent numbering indent level

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
(Documentation change only)
```

* Test the code
See steps below to replicate - documentation change only
```
```

## What to Check
You can repeat running app registration PowerShell script and editing signInAudience (then saving) in app registrations blade as previously indicated in doc, you will see error re: accesstokenAcceptedVersion (by default it is set to null)
* ...

## Other Information
Link to documentation:
https://docs.microsoft.com/en-us/azure/active-directory/develop/reference-app-manifest
